src/interfaces/NveComponent.interface.ts

```diff
@@ -0,0 +1,5 @@
+
+export interface INveComponent {
```

Comment:
Her trenger vi dokumentasjon, både av interfacet og testId-propertyen. Er det lurt å skrive hvordan man må annotere denne for at den skal bli en property og attributt også?